plugin.video.srgssr_ch_replay/resources/settings.xml

```diff
@@ -1,17 +1,90 @@
-<settings>
-	<category label="30100">
-		<setting id="consumerKey" type="text" label="30101" />
-		<setting id="consumerSecret" type="text" label="30102" />
-		<setting id="subtitlesEnabled" type="bool" label="30103" default="false" />
-		<setting id="consumerKeySubtitles" type="text" label="30104" enable="eq(-1,true)" subsetting="true" />
-		<setting id="consumerSecretSubtitles" type="text" label="30105" enable="eq(-2,true)" subsetting="true" />
-	</category>
-	<category label="30200">
-		<setting id="numberOfShowsPerPage" type="number" label="30201" default="10" />
-		<setting id="enableInputstreamAdaptive" type="bool" label="30202" default="false" />
-	</category>
-	<category label="30900">
-		<setting id="showInactiveShows" type="bool" label="30901" default="false" />
-		<setting id="disableLetterMenu" type="bool" label="30902" default="false" />
-	</category>
-</settings>
\ No newline at end of file
+<?xml version="1.0" ?>
+<settings version="1">
+	<section id="plugin.video.srgssr_ch_replay">
+		<category help="" id="srg ssr api" label="30100">
+			<group id="1">
+				<setting help="" id="consumer_key" label="30101" type="string">
+					<level>0</level>
+					<default/>
+					<constraints>
+						<allowempty>true</allowempty>
+					</constraints>
+					<control format="string" type="edit">
+						<heading>30101</heading>
+					</control>
+				</setting>
+				<setting help="" id="consumer_secret" label="30102" type="string">

```

Comment:
The keys have changed so we need a migration temporary migration script for all these keys consumerKey -> consumer_key so the users do not have to re-input them. I suggest something like if consumer_key is empty but consumerKey is not then we set it into consumer_key the first when the add-on is started. We should do this for all the settings and the migration script can then be removed again

Reply:
Didn't thought about that, that seems a good idea to me :+1: